### ISSUE TITLE
Add self-signed cert to NGINX

### ISSUE THREAD

**Issue by mikeanthonywild** [sdcc-sh/sdcc-ansible#34]
As a stop-gap until there's time to get Let's Encrypt up and running, a self-signed certificate should be generated and added to NGINX so there's some encryption for services with logons (e.g. subsonic).

**Comment by mikeanthonywild** [comment]
Closed by #37.

**Comment by mikeanthonywild** [comment]
`req.conf`:

```ini
[req]
distinguished_name = req_distinguished_name
x509_extensions = v3_req
prompt = no
[req_distinguished_name]
O = Socially Distant Computer Club
CN = www.sdcc.dev
[v3_req]
keyUsage = keyEncipherment, dataEncipherment
extendedKeyUsage = serverAuth
subjectAltName = @alt_names
[alt_names]
DNS.1 = sdcc.dev
DNS.2 = subsonic.sdcc.dev
```
